server/src/internalClusterTest/java/org/elasticsearch/search/fieldcaps/CCSFieldCapabilitiesIT.java

```diff
@@ -266,4 +268,16 @@ public void testReturnAllLocal() {
             }
         }
     }
+
+    public void testSummarizeFailures() {
+        var response = client().prepareFieldCaps("remote_cluster:no-such-index").setFields("*").setSummarizeFailures(true).get();
+
+        assertThat(response.getIndexResponses(), hasSize(0));
+        assertThat(response.getFailures(), hasSize(1));
+        var failure = response.getFailures().getFirst();
+        var cause = failure.getException().getCause();
+        assertThat(cause, isA(IndexNotFoundException.class));
+        assertThat(cause.getMessage(), equalTo("no such index [no-such-index]"));
+        assertThat(failure.getIndices(), arrayContaining("remote_cluster:no-such-index")); // this contains actual remote
```

Comment:
It would be nice if we could adapt it to:
* contain all indices not found failures
* be able to report indices not found and security exceptions at once
* collect unknown remotes exceptions (so that a caller could handle them according to skip unavailable)